homeworks/sargis_vardanyan/HelloWorldMaven/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
the target folder is autogenerated one, and should be added to ignored list in .gitignore

Reply:
after changing .gitignore as mentioned, I tried 
`git add .` 
and then, commit, push, but the target folder still exists